cartModel.js

```diff
@@ -0,0 +1,79 @@
+module.exports = CartModel = function () {
```

Comment:
Please use native functions or lodash functions instead of the imperative approaches